sdk/appconfiguration/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/manager/models/FeatureVariant.java

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.feature.manager.models;
+
+import java.util.LinkedHashMap;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A variant of a feature.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public final class FeatureVariant {
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("default")
+    private Boolean isDefault = false;
+
+    @JsonProperty("configuration-reference")
+    @JsonAlias({"configurationReference", "configuration-reference"})
+    private String configurationReference;
+
+    @JsonProperty("assignment-parameters")
+    @JsonAlias({"assignment-parameters", "assignmentParameters"})
```

Comment:
I don't think we need `configurationReference` and `assignmentParameters` in the @JsonAlias

Reply:
How about this:

```java
@JsonProperty("assignment-parameters")    
@JsonAlias("assignmentParameters")
private LinkedHashMap<String, Object> assignmentParameters;
```

I suppose this means `assignmentParameters` will be serialized and deserialized as **"assignment-parameters"**, and it will also take **"assignmentParameters"** when do deserialization.